GVFS/GVFS.Service/Program.Mac.cs

```diff
@@ -15,10 +18,42 @@ public static void Main(string[] args)
 
             using (JsonTracer tracer = new JsonTracer(GVFSConstants.Service.ServiceName, GVFSConstants.Service.ServiceName))
             {
-                GVFSService service = new GVFSService(tracer);
+                CreateService(tracer, args).Run();
+            }
+        }
 
-                service.RunWithArgs(args);
+        private static GVFSService CreateService(JsonTracer tracer, string[] args)
+        {
+            string serviceName = args.FirstOrDefault(arg => arg.StartsWith(GVFSService.ServiceNameArgPrefix, StringComparison.OrdinalIgnoreCase));
+            if (serviceName != null)
+            {
+                serviceName = serviceName.Substring(GVFSService.ServiceNameArgPrefix.Length);
+            }
+            else
+            {
+                serviceName = GVFSConstants.Service.ServiceName;
             }
+
+            GVFSPlatform gvfsPlatform = GVFSPlatform.Instance;
+
+            string logFilePath = Path.Combine(
+                gvfsPlatform.GetDataRootForGVFSComponent(serviceName),
+                GVFSConstants.Service.LogDirectory);
+            Directory.CreateDirectory(logFilePath);
+
+            tracer.AddLogFileEventListener(
+                GVFSEnlistment.GetNewGVFSLogFileName(logFilePath, GVFSConstants.LogFileTypes.Service),
+                EventLevel.Verbose,
```

Comment:
Usually log file listeners are created with `EventLevel.Informational` level, why was this created with `EventLevel.Verbose`?